src/components/InfoIcon/InfoIcon.tsx

```diff
@@ -1,11 +1,12 @@
 import * as React from 'react'
 import classnames from 'classnames'
+import { omit } from 'decentraland-commons/dist/utils'
 import styles from './InfoIcon.module.css'
 import { Props } from './InfoIcon.types'
 
 export default class InfoIcon extends React.PureComponent<Props> {
   render() {
     const { className } = this.props
-    return <i className={classnames(styles.info, className)} />
+    return <i {...omit(this.props, ['className'])} className={classnames(styles.info, className)} />
```

Comment:
is the omit necessary here? as we're  overwriting it with the next prop

Reply:
Yeah, you're right, it gets overwritten, removing the omit.